README.rst

```diff
@@ -0,0 +1,150 @@
+Cromwell-tools
```

Comment:
Just to clarify, does sphinx only work with `.rst` and not `.md`?

Reply:
I believe there are plugins that allow it to work with `.md`. I haven't used them myself, but it would be nice to figure out if it's possible!

Reply:
Yes, it works for both rst and md files. The most popular extension to enbale md support for sphinx is recommonmark, check here for example: https://github.com/rtfd/recommonmark/tree/master/docs

Reply:
I remember the reason I didn't use the recommonmark for this PR was that using md files in sphinx is more like embedding rst into markdown blocks at the first glance to me. I didn't look into it deeper with a feeling of "not worth it". But I'm happy to do the research about md support for Lira in the next PR :)